ghost/members-api/lib/controllers/MemberController.js

```diff
@@ -41,6 +49,13 @@ module.exports = class MemberController {
             return res.end('No Permission.');
         }
 
+        const blockedEmailDomains = this._settingsCache.get('all_blocked_email_domains');
+        const emailDomain = req.body.email.split('@')[1]?.toLowerCase();
+        if (emailDomain && blockedEmailDomains.includes(emailDomain)) {
+            res.writeHead(400);
```

Comment:
I tried adding but for some reason it completely crashes Express / Node and I couldn't get to the bottom of why it's happening.

```
throw new errors.BadRequestError({
                message: tpl(messages.blockedEmailDomain)
            });
```

Reply:
A `throw` would need a `try/catch` block within the method (cf. `throw` example a couple of lines below in the method).

I think your approach with writing the response with a 400 directly in that case is good 👍